src/documents/RemoteModelAPI.md

```diff
@@ -0,0 +1,316 @@
+---
+title: RemoteModel API
+layout: default
+category: Infusion
+---
+
+`fluid.remoteModelComponent` builds on top of [`fluid.modelComponent`](ComponentGrades.md) with the purpose of providing a buffer between a local and
+remote model that are attempting to stay in sync. For example a local model is being updated by user interaction, this is sent back
+to a remote server, which in turn tries to update the local model. If additional user actions occur during the roundtrip, an
+infinite loop of updates may occur. `fluid.remoteModelComponent` solves this by restricting reading and writing to a single request at a time,
+waiting for one request to complete before operating the next.
+
+## Supported Events
+
+In addition to the standard `fluid.modelCompnent` events, `fluid.remoteModelComponent` add the following:
+
+* `onFetch`,
+* `onFetchError`,
+* `afterFetch`,
+* `onWrite`,
+* `onWriteError`,
+* `afterWrite`
+
+For information on the different types of events, see [Infusion Event System](InfusionEventSystem.md).
```

Comment:
We don't really have different "types" of events any more - all the non-core varieties are deprecated. Should instead say something like, for information on how events work and how to configure listeners to them, see xxxxx